doc/organization.md

```diff
@@ -1,14 +1,18 @@
 # Organization
 
-- Front-End Development:
+## Front-End Development:
+Front-End Development refers to the gui components of the project. The look and feel of the project is handled in this role. 
 
   -- Jonasty
```

Comment:
Should probably change this to his real name (Jonathan Hasty) to match with the other roles.